### PR TITLE
pages.nl/common/*: add Dutch translation

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.nl/common/fgrep.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# fgrep
+
+> Zoek naar strings in bestanden. Gelijk aan `grep -F`.
```
Comment (kbdharun) [inline pages.nl/common/fgrep.md:3]:
```suggestion
> Zoek naar strings in bestanden.
> Gelijk aan `grep -F`.
```

File: pages.nl/common/vimdiff.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# vimdiff
+
+> Open twee of meer bestanden in `vim` en toon de verschillen.
+> Bekijk ook: `vim`, `vimtutor`, `nvim`.
+> Meer informatie: <https://www.vim.org>.
+
+- Open twee bestanden en toon de verschillen:
+
+`vimdiff {{pad/naar/bestand1}} {{pad/naar/bestand2}}`
```
Comment (kbdharun) [inline pages.nl/common/vimdiff.md:9]: Can you update the same in English page too?

**Review by kbdharun — APPROVED** [review]

File: pages.nl/common/vimdiff.md (hunk 2 above)
Comment (kbdharun) [reply pages.nl/common/vimdiff.md:9]: > What exactly? It is/was a copy of the English page

The English page doesn't have the path placeholder, I would suggest adding the same over there too.

File: pages.nl/common/egrep.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Zoek voor een patroon in een bestand:
+
+`egrep "{{zoekpatroon}}" {{pad/naar/bestand}}`
+
+- Zoek voor een patroon in meerdere bestanden:
+
+`egrep "{{zoekpatroon}}" {{pad/naar/bestand1}} {{pad/naar/bestand2}} {{pad/naar/bestand3}}`
+
+- Zoek in `stdin` voor een patroon:
```
Comment (leonvsc) [inline pages.nl/common/egrep.md:14]:
```suggestion
- Zoek in `stdin` naar een patroon:
```

Same for this.

File: pages.nl/common/vimtutor.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# vimtutor
+
+> Vim tutor leert de basis `vim` commando's.
+> Bekijk ook: `vim`, `vimdiff`, `nvim`.
+> Meer informatie: <https://manned.org/vimtutor>.
+
+- Start de vim tutor voor de opgegeven taal (en, fr, de, ...):
+
+`vimtutor {{language}}`
```
Comment (leonvsc) [inline pages.nl/common/vimtutor.md:9]:
```suggestion
`vimtutor {{taal}}`
```

**Review by leonvsc — DISMISSED** [review]
Some small changes and suggestions left.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**